compiler/tools/fpp-to-dict/test/scripts/run.sh

```diff
@@ -53,11 +53,16 @@ validate_json_schema()
   # Check to see if the dictionary JSON schema is valid
   if which python3 > /dev/null 2>&1
```

Comment:
This looks good, but I think we can clean up the logic. I don't think we need two nested if statements: the outer one tests for `python3`, but if there's no `python3`, then the inner one will fail. So I'd delete the outer one and keep the inner one.